Missing objective data for quest 26050 - Goggle Boggle (Arathi Highlands, Cataclysm Classic)

<!-- READ THIS FIRST

Hello, thanks for taking the time to report a bug!

Before you proceed, please verify that you're running the latest version of Questie. The easiest way to do this is via the Curseforge client, but you can also download the latest version here: https://www.curseforge.com/wow/addons/questie

Questie is one of the most popular Classic WoW addons, with over 30 million downloads. However, like almost all WoW addons, it's built and maintained by a team of volunteers.

If you'd like to help, please consider making a donation. You can do so here: https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=aero1861%40gmail%2ecom&lc=CA&item_name=Questie%20Devs&currency_code=USD&bn=PP%2dDonationsBF%3abtn_donate_LG%2egif%3aNonHosted

You can also help as a tester, developer or translator. To do so please join the Questie Discord here https://discord.gg/fYcQfv7

-->


## Description
<!-- Explain in detail what problem you encountered. Is a quest not showing or showing again or an objective not showing or a spawn missing? -->

Hey folks! While continuing my questing through Arathi Highlands, I ran into another missing objective data error on Goggle Boggle:

[ERROR] Missing objective data for quest 26050 Defend Professor Phizzlethorpe

## Wowhead link
<!-- If this issue is about a quest/npc/item/... then please include a link to it from Wowhead: https://wowhead.com/classic or https://wowhead.com/wotlk or https://wowhead.com/cata -->

https://www.wowhead.com/cata/quest=26050/goggle-boggle

## Screenshots
<!-- If you can, add a screenshot to help explaining the problem (e.g. of your map where a quest is not shown but the NPC has an ! above him). Simply drag and drop the image in this input field, no need to upload it to any other image platform. -->

Questie Error:
![image](https://github.com/user-attachments/assets/feb3e38c-701c-4e26-8684-4a6d65987163)

Quest Description:
![image](https://github.com/user-attachments/assets/dfc584fd-3d77-47d5-a062-fccfa2ab78e9)

## Questie & WoW version
<!--
Which version of Questie are you using? You can find it by:
- 1. Hovering over the Questie Minimap Icon
- 2. looking at your Questie.toc file (open it with any text editor).
It looks something like this: "v5.9.0" or "## Version: 5.9.0".

Please also state if your quest issue is about Classic Era/SoM or Wotlk.
-->

Questie 10.3.3
WoW 4.4.0 (Cataclysm Classic)

--- a/Database/Corrections/cataQuestFixes.lua
+++ b/Database/Corrections/cataQuestFixes.lua
@@ -3560,6 +3560,9 @@ function CataQuestFixes.Load()
         [26043] = { -- BEWARE OF CRAGJAW!
             [questKeys.preQuestSingle] = {26004},
         },
+        [26050] = { -- Goggle Boggle
+            [questKeys.triggerEnd] = {"	Defend Professor Phizzlethorpe", {[zoneIDs.ARATHI_HIGHLANDS]={{33.80,80.40}}}},
+        },
         [26056] = { -- The Wavespeaker
             [questKeys.nextQuestInChain] = 26065,
         },

--- a/Database/Corrections/cataQuestFixes.lua
+++ b/Database/Corrections/cataQuestFixes.lua
@@ -3561,7 +3561,7 @@ function CataQuestFixes.Load()
             [questKeys.preQuestSingle] = {26004},
         },
         [26050] = { -- Goggle Boggle
-            [questKeys.triggerEnd] = {"	Defend Professor Phizzlethorpe", {[zoneIDs.ARATHI_HIGHLANDS]={{33.80,80.40}}}},
+            [questKeys.triggerEnd] = {"Defend Professor Phizzlethorpe", {[zoneIDs.ARATHI_HIGHLANDS]={{33.80,80.40}}}},
         },
         [26056] = { -- The Wavespeaker
             [questKeys.nextQuestInChain] = 26065,

